db/seeds.rb

```diff
@@ -14,51 +10,241 @@
+  description: "What’s in a name? Well, we’ve always been fans of the opascope, a beautiful tool for projecting handwriting and finely rendered artwork onto a larger surface. We could blab for hours about its quaint design, its bulky contours, its place in our childhood classrooms. But for expediency’s sake, let’s put it this way: The opascope is a tool for taking careful craftsmanship and opening it up for everyone to access and enjoy. Sound familiar? Available only in the Bay Area and online, Opascope Espresso is a refreshing addition to a lineup once dominated by dense, chocolatey selections. It yields an effervescent shot, packed with stripes of tropicalia.",
+  image_url: "https://blue-bottle-cms.global.ssl.fastly.net/hbhhv9rz9/image/upload/c_thumb,h_576,w_576/v1526404899/uwboy5luvxzmrttbe3ml.jpg",
+  quantity: rand(1..20),
+  price: rand(10..20),
+  created_at: rand(1.0..30.0).days.ago
+)
 
-item_1 = create(:item, user: merchant_1)
-item_2 = create(:item, user: merchant_2)
-create_list(:item, 10, user: merchant_1)
+item_11 = merchants[7].items.create(
+  name: "Bodum Chambord 17oz French Press",
+  description: "This excellent single-person French press from Bodum includes chrome feet, black bakelite plastic knob and handle, and special Bodum-designed insulated lid. This three-tasse design has better heat retention than the skinny three-tasse size.",
+  image_url: "https://blue-bottle-cms.global.ssl.fastly.net/hbhhv9rz9/image/upload/c_thumb,h_576,w_576/v1490999589/xzgrd3bqvysyqxykvibs.jpg",
+  quantity: rand(1..20),
+  price: rand(30..40),
+  created_at: rand(1.0..30.0).days.ago
+)
 
-inactive_item_1 = create(:inactive_item, user: merchant_1)
-inactive_item_2 = create(:inactive_item, user: inactive_merchant_1)
+item_12 = merchants[8].items.create(
+  name: "AeroPress",
+  description: "Because it was conceived just 38.5 miles from our Oakland roastery by Aerobie (those wonderful folks who created an improbably long-flying disc), we'd like to humbly assert that the AeroPress is our first locally-produced piece of merchandise. It’s a peculiar and lovely device – easily the most durable and portable option for brewing quality coffee. It produces a cup that’s thick and focused, but still quite nuanced.",
+  image_url: "https://blue-bottle-cms.global.ssl.fastly.net/hbhhv9rz9/image/upload/c_thumb,h_576,w_576/v1486399954/u06l7rfl6noxg1t4reim.jpg",
+  quantity: rand(1..20),
+  price: rand(30..40),
+  created_at: rand(1.0..30.0).days.ago
+)
 
-Random.new_seed
-rng = Random.new
+inactive_item_1 = create(:inactive_item, user: merchants[9])
+inactive_item_2 = create(:inactive_item, user: merchants[10])
+
+
+# Order generation of all types
+user_count = User.where(role: 0).count
+item_count = Item.count
 
 # pending order, none fulfilled
-order = create(:order, user: user)
-create(:order_item, order: order, item: item_1, order_price: 1, quantity: 1)
-create(:order_item, order: order, item: item_2, order_price: 2, quantity: 1)
+n = 1
```

Comment:
This whole section is brilliant man.  Great use of the rand operator to spice up the statistics!